test/integration/build_to_buildruns_test.go

```diff
@@ -412,38 +409,6 @@ var _ = Describe("Integration tests Build and BuildRuns", func() {
 			Expect(err).To(BeNil())
 			Expect(ownerReferenceNames(br.OwnerReferences)).ShouldNot(ContainElement(buildObject.Name))
 
-		})
-		It("does not deletes the buildrun if the annotation is removed", func() {
```

Comment:
The equivaluent would be to change the field to `nil`, or to remove the overall `retention` object?

Reply:
added.